raiden/network/blockchain_service.py

```diff
@@ -102,8 +102,11 @@ def get_block_header(self, block_number: int):
         return self.client.web3.eth.getBlock(block_number, False)
 
     def next_block(self) -> int:
+        target_block_number = self.block_number() + 1
+        self.wait_until_block(target_block_number=target_block_number)
+
+    def wait_until_block(self, target_block_number):
```

Comment:
The PR says "increased sleep for wait until block" but I don't see anything like that here though.